OneCommunityGlobal/HighestGoodNetworkApp
# Yubo fix no popup warning for empty media url

# Description
A small fix in `EditLinkModal` 

## Related PRS (if any):
It's a small fix in PR #1236 

## Main changes explained:
- Add a condition to determine if the input of **Media Folder Link** was empty
- Add an update to the `ref` that stores the original **Media Folder Link** after link submitted. Now, it always compare the link you are typing with the updated link with no need to refresh the page. (Thanks to @GaryB93 pointed out in PR #1236 ) 

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as admin user -> go to other user's profile page -> click link **Edit** under picture.
### For 'no popup warning'
- Try to change the `Media Folder` link and see the popup won't show up if current user has empty **Media Folder Link** (you can delete to empty to test)
### For 'comparing updated link'
- Update one user's **Media Folder Link**  (this case, you should find a user that has **Media Folder Link**, you can also add one before testing). After you click 'update' button, it should close automatically.
- click the `Edit` button again and make some changes to the input of ** Media Folder Link** and see the warning is indicating the updated link you made. Because before the changes, the warning will show the link before you updated.

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/57503876/24707ab9-a111-4183-8460-439f51494020
